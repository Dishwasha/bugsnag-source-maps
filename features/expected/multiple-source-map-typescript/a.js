function hello() {
    console.log('hello from a');
}
export default hello;
//# sourceMappingURL=a.js.map
