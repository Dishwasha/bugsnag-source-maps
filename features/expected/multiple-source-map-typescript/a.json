{
    "version": 3,
    "file": "a.js",
    "sourceRoot": "",
    "sources": [
        "src/lib/a.ts"
    ],
    "names": [],
    "mappings": "AAAA,SAAS,KAAK;IACZ,OAAO,CAAC,GAAG,CAAC,cAAc,CAAC,CAAA;AAC7B,CAAC;AAED,eAAe,KAAK,CAAA",
    "sourcesContent": [
        "function hello(): void {\n  console.log('hello from a')\n}\n\nexport default hello\n"
    ]
}
